client/src/components/Form/InputDisplay/InputDisplay.tsx

```diff
@@ -55,14 +70,14 @@ const InputDisplay = (props) => {
                 />)
                 : null
             }
-            <Col xm={12} lg={6}>
+            <Col xs={12} lg={6}>
                 <div className="display-container">
                     <div className="display-label">
                         <p className="display-title">{inputTitle}</p>
                         <button
                             className="display-button visible-button"
-                            onClick={ () => inputVisibility(inputId) }
-                            disabled ={ toDisable && toDisable.has('visibility') }
+                            onClick={ () => inputVisibility && inputVisibility(inputId) }
```

Comment:
We should reconsider the names of these variables to express what they really do or cointain. like setInputVisibility